content-copy-tool/README.md

```diff
@@ -51,6 +64,22 @@ cd ~/Documents/content-copy-tool
 through Finder). Note: this command will
 bring you to the top level directory of the tool from anywhere in a terminal.
 
+#### virtualenv (optional but recommended)
+
+For first time create a new virtualenv environment in the content-copy-tool folder if it's not existing:
+```bash
+virtualenv env
```

Comment:
Will this pick python2 by default? Mine seems to take the system installed python by default. What about something similar to the following:
```suggestion
virtualenv env -p $(which python2.7)
```

Reply:
oh, you can really use `virtualenv` with python3 with the same command? For some unknown reason (but I'm also not using bash and virtualenv often) I've never got it to work with python3. I will add your modifications